oShare/MultipeerConnectivityManager.swift

```diff
@@ -4,7 +4,7 @@ class MultipeerConnectivityManager: NSObject {
 	
 	weak var delegate: MultipeerConnectivityManagerDelegate?
 
-	var peer: MCPeerID
+	var localPeer: MCPeerID
```

Comment:
This change better disambiguates the reason for this property. This is _us_, and not the remote `peer`.